data/json/items/comestibles/raw_fruit.json

```
@@ -764,16 +764,16 @@
     "comestible_type": "FOOD",
     "symbol": "%",
     "quench": 24,
-    "calories": 496,
+    "calories": 130,
     "description": "A very sweet and soft tropical fruit.",
     "price": 118,
     "price_postapoc": 300,
     "material": [ "fruit" ],
-    "volume": "400 ml",
+    "volume": "594 ml",
     "fun": 5,
     "flags": [ "SMOKABLE" ],
     "smoking_result": "dry_fruit",
-    "vitamins": [ [ "vitC", 205 ], [ "calcium", 4 ], [ "iron", 4 ] ]
+    "vitamins": [ [ "vitC", "185100 μg" ], [ "iron", "800 μg" ], [ "calcium", "60800 μg" ] ]
```

This is also a valid format, if you think it's more readable.
```suggestion
    "vitamins": [ [ "vitC", "185 mg 100 μg" ], [ "iron", "800 μg" ], [ "calcium", "60 mg 800 μg" ] ]
```

---

I wasn't sure if it worked or not so I went with the safer bet. The first alternative I've gone through uses less signs so I think it's marginally better, but if your vitamin display PR is going to show this in the format I used I might need to rethink that.

---

The vitamin display doesn't care about what you put here more than that it defines the "units" of the vitamin. It will also truncate to the highest unit (this would show as 185 mg, 800 ug, 60mg) - once the rounding of conversion has done (it'll convert weight -> units -> weight, which will lose some info).

---

> The vitamin display doesn't care about what you put here more than that it defines the "units" of the vitamin. It will also truncate to the highest unit (this would show as 185 mg, 800 ug, 60mg) - once the rounding of conversion has done (it'll convert weight -> units -> weight, which will lose some info).

In this case I'll probably stick to defining it the way I'm doing now. I find it easier to read, so unless people would consider this too confusing I'll likely stick to what's comfortable for me.

---

I just wanted to point it out in case you were unaware.
I think we've talked about everything that needs talking about here.